assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -77,6 +77,9 @@ export default function SettingsView() {
 	const canUseSnippet = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).getCanUseSnippet()
 	);
+	const useGA4Snippet = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getUseSnippet()
```

Comment:
We still use the `ga4setup` feature flag in this component, so let's call the `getUseSnippet` selector only if the `isGA4Enabled` is truthy.

Reply:
👍